parseTorrentName.ts

```diff
@@ -199,7 +199,11 @@ const processFolder = (
   const folderPath = `${currentDirectory}/${folderName}`;
   const files = fs.readdirSync(folderPath);
   files.forEach((file) => {
-    movedFileInfos.push(processFile(folderPath, file, targetDirectory));
+    if (isFile(`${folderPath}/${file}`)) {
+      movedFileInfos.push(processFile(folderPath, file, targetDirectory));
+    } else {
+    movedFileInfos = processFolder(folderPath, file, targetDirectory);
```

Comment:
Ca marche pas. Si tu as un fichier qui est un film et un folder one next to the other tu excrase le movedFileInfos la deuxieme fois que tu passes la loop